### ISSUE TITLE
Black screen on Elgato Cam Link 4K and generic HDMI to UVC adapter

### ISSUE THREAD

**Issue by jeremyherbert** [Martoni/HdmiCore#7]
Hi,

Thanks for sharing this project! I'm currently running it on the Tang Nano 9k and it works great on my HDMI LCD monitor, but I can't get it to show anything on HDMI capture devices. I have tried with an Elgato Cam Link 4K, as well as a generic HDMI to UVC adapter: https://www.adafruit.com/product/4669 (these are all over ebay/aliexpress too)

On the Cam Link is an LED which indicates that it is capturing, and when I connect the Tang Nano the LED remains off. Also, when I connect the device to a HDMI switcher, it doesn't detect that a video input has been connected. On the generic UVC adapter, it seems like I just get a black screen, though I am unsure exactly if it can detect the video stream or it just outputs a black screen by default.

Have you tested this on any HDMI capture devices? I haven't modified the project at all, I just followed the instructions in the readme, then built and flashed it with the gowin IDE and openFPGAloader. FWIW, I also flashed the picotiny example which comes with the Tang Nano 9k, and it also fails to capture.

**Comment by Martoni** [comment]
Hi @jeremyherbert,
Have you tryied the project with a standard HDMI screen ?
I never tested it with a HDMI capture devices, but it's an interesting test.
If I can found this device I will test.

**Comment by jeremyherbert** [comment]
Yes, I have, my HDMI monitor works fine. I was just hoping to tinker with HDMI but without giving up a whole screen :)

You can find them for quite cheap on Aliexpress by searching for "HDMI capture USB", for example: [here](https://www.aliexpress.com/item/4001010505067.html?spm=a2g0o.productlist.0.0.46962841pdHJiT&algo_pvid=4bf7ab9c-ee4a-4c68-8e64-257e9345274f&algo_exp_id=4bf7ab9c-ee4a-4c68-8e64-257e9345274f-1&pdp_ext_f=%7B%22sku_id%22%3A%2210000013437676828%22%7D&pdp_pi=-1%3B11.44%3B-1%3B-1%40salePrice%3BAUD%3Bsearch-mainSearch).
